target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -25,44 +25,53 @@ edgecore,ecw5410)
 nec,wg2600hp)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy1tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy0tpt"
-	ucidef_set_led_switch "wan" "WAN" "green:active" "switch0" "0x2"
+	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan"
```

Comment:
If you are OK, to make it behave the same as before the change, I would like you to add ```"link"``` for WG2600HP. (in the stock firmware, "active" LED doesn't blink)

```
diff --git a/target/linux/ipq806x/base-files/etc/board.d/01_leds b/target/linux/ipq806x/base-files/etc/board.d/01_leds
index 6a429bf03d..1bbee4676e 100644
--- a/target/linux/ipq806x/base-files/etc/board.d/01_leds
+++ b/target/linux/ipq806x/base-files/etc/board.d/01_leds
@@ -24,7 +24,7 @@ edgecore,ecw5410)
 nec,wg2600hp)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy1tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy0tpt"
-	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan"
+	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan" "link"
 	;;
 nec,wg2600hp3)
 	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan"

```